### ISSUE TITLE
Vim clipboard should be persistent

### ISSUE THREAD

**Issue by M0M097** [cdepillabout/termonad#179]
If I use Vim in xterm I can copy text to various clipboards, for example highlighting copies to the default X clipboard and an additional "+y to the "normal" clipboard. If I now leave vim I can paste the content with middle click from the X clipboard or shift-insert from the normal one.

The same works with termonad as long as vim is running: I can copy text by highlighting and paste it into another instance of termonad. 

But as soon as I quit Vim this buffer is dead and I can't paste its content anymore. The xterm way is very convenient if one wants to copy content from one file to another or into the terminal.

I don't know if there is an easy way to implement this request and don't know how xterm or other terminals achieve that but think this should be added since it boosts productivity quite a bit.

Edit: this only occurs in minimal window managers, I tested fluxbox and xmonad. In full DE like KDE the clipboards seem to be handled by the DE.

**Comment by M0M097** [comment]
Ok, it was my mistake since I did not understand how clipboards work. The issue was the following: 

to remember the content of a clipboard when the application from which it was copied is closed a clipboard manager is needed. There are several types of clipboard managers:

1. Big DE like KDE ship with an integrated clipboard manger therefore one can paste stuff regardless if the application from which it was copied is running or not. Minimal WMs like fluxbox or xmonad do not ship with a integrated clipboard manager. 

2. Some terminal emulators have a clipboard manger as well, for example the mentioned xterm. So one can paste stuff from a closed application that was running inside this terminal. Termonad does not ship with a clipboard manager and if it is used in a minimal WM there is no clipboard manger at all. So my request was equivilant to "add a integrated clipboard manager to termonad" However after I understood how it works I don't think thats a good idea since there is a third option:

3. One can use a dedicated clipboard manager. For example clipit is very light and works perfectly, one just need to start it as a daemon in the .xinitrc file. This way one follows the UNIX philosophy that a program should do one task only.

To sum it up: I don't think there is a need for an internal clipboard manger.

**Comment by cdepillabout** [comment]
@M0M097 Thanks for summing this up!

I think this is a very good summary of the situation for folks that use a minimal window manager.
